src/components/TagWrapper/PTagWrapper.vue

```diff
@@ -11,12 +11,14 @@
     </slot>
 
 
-    <div ref="overflowTag" class="p-tag-wrapper__tag-overflow" :class="classes.overflowTag" :title="hiddenText">
-      <slot name="overflow-tags" :overflowed-children="overflowCount">
-        <PTag>
-          +{{ overflowCount }}
-        </PTag>
-      </slot>
+    <div ref="overflowTag" class="p-tag-wrapper__tag-overflow" :class="classes.overflowTag">
+      <p-tooltip :text="hiddenText">
```

Comment:
This assumes a global installation of all components; can you update this to match the import style in use elsewhere? Meaning import `PTooltip` and use pascal-case component syntax